bemserver_ui/static/scripts/modules/views/timeseries/dataExplore.js

```diff
@@ -157,7 +223,25 @@ export class TimeseriesDataExploreView {
             flaskES6.urlFor(`api.timeseries_data.retrieve_multiple_data`, urlParams),
             (data) => {
                 this.#chart.setDownloadCSVLink(flaskES6.urlFor(`timeseries_data.download_multiple`, urlParams));
-                this.#chart.load(data, this.#tsDataStatesSelectElmt.options[this.#tsDataStatesSelectElmt.selectedIndex].text, this.#timezonePickerElmt.tzName);
+
+                let options = {
+                    subtitle: this.#tsDataStatesSelectElmt.options[this.#tsDataStatesSelectElmt.selectedIndex].text,
+                    timezone: this.#timezonePickerElmt.tzName,
+                    series: {},
+                };
+
+                for (let [index, tsId] of Object.entries(this.#tsSelector.selectedItemIds)) {
+                    let tsName = this.#tsSelector.selectedItemNames[index];
+
+                    options.series[tsName] = {
+                        position: document.getElementById("position-" + this.#tsSelector.selectedItemIds[index]).value,
```

Comment:
I don't understand this, because i want the name of the timeseries for `option.series[tsName]` and `tsId` return just the index of the timeseries in the array list for example `1` so it doesn't work in this case.